### PR TITLE
IntelliJ-based IDEs: debugging is now supported in IDEA Ultimate

### PR DESCRIPTION
**Pull request by MarinaKalashina**
Please update the info on debugger support in IntelliJ-based IDEs.
Debugger is now supported in IntelliJ IDEA Ultimate version 2020.1 and later. The initial announcement in the plugin's changelog: https://intellij-rust.github.io/2020/04/27/changelog-121.html

### CODE REVIEW

File: index.jade [hunk 1]
```diff
@@ -288,7 +288,7 @@ html(lang="en")
               a(href="https://intellij-rust.github.io/features/") code insight features
               |. You can work with Cargo commands and run Clippy or Rustfmt without leaving the IDE.
             p
-              | Debugger and profiler are available in CLion.
+              | Debugger is available in CLion and IntelliJ IDEA Ultimate.
```
Comment (Undin) [inline index.jade:291]: @MarinaKalashina Can we keep the note about CLion profiler?

Comment (MarinaKalashina) [reply index.jade]: Sure, thanks for pointing

**Comment by contradictioned** [comment]
Great, thank you!

**Comment by MarinaKalashina** [comment]
Thank you, but I'm afraid the merged text requires some corrections, I created a new PR: https://github.com/contradictioned/areweideyet/pull/97

**Pull request merged**